api/swagger.yml

```
@@ -2776,6 +2776,46 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Role"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        404:
+          $ref: "#/components/responses/NotFound"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.38.0)

[error] too many spaces inside empty brackets

(brackets)